yarn.lock

```diff
@@ -3,9 +3,9 @@
 
 
 "@babel/cli@^7.10.1":
-  version "7.14.8"
-  resolved "https://registry.yarnpkg.com/@babel/cli/-/cli-7.14.8.tgz#fac73c0e2328a8af9fd3560c06b096bfa3730933"
-  integrity sha512-lcy6Lymft9Rpfqmrqdd4oTDdUx9ZwaAhAfywVrHG4771Pa6PPT0danJ1kDHBXYqh4HHSmIdA+nlmfxfxSDPtBg==
+  version "7.15.4"
+  resolved "https://registry.yarnpkg.com/@babel/cli/-/cli-7.15.4.tgz#00e21e192b738dec7900c8bb36270e377217c0a4"
```

Comment:
I just noticed now that this file is being changed, this is wrong, please revert any change to yarn.lock, I suggest rebasing on the latest master, updating your packages by running `yarn` and leave this file alone.

Reply:
I reverted the changes to yarn.lock